### ISSUE TITLE
Virtual Scroll and large list

### ISSUE THREAD

**Issue by berrg** [quasarframework/quasar#8472]
**Describe the bug**
When working with large list the scroll content suddenly becomes empty when scrolling past a certain point.

**Codepen/jsFiddle/Codesandbox (required)**

https://codepen.io/morten-berg/pen/BaQpJMB

**To Reproduce**
This is a list with 700.000 items, when scrolling past element ~690.000, the view becomes empty.  When using chrome inspector you can see the element, but is not visible.

**Comment by pdanpdan** [comment]
Yes, I'll have to think of some way to work around this - there is a limit on the height of an element after which the browsers start behaving as they should not.
It looks like chrome's engine on linux has this limit around 1048600 * 32px

**Comment by berrg** [comment]
Could it be a solution to use padding-top and padding-bottom on main element instead of using divs for making the empty space?

**Comment by pdanpdan** [comment]
no, it's about JS outputting big numbers in exponential notation and losing precision.

**Comment by pdanpdan** [comment]
Ok, I found a solution to make it work up to 9,999,999,999,999,998 - now let's see if it works for CSS also :)

**Comment by pdanpdan** [comment]
Yep, not too much chances to improve something is a short time / without a different approach that would decrease the precision of scroll.
You can try something like this now (paginate the data):
https://codepen.io/pdanpdan/pen/JjbEvYW?editors=1010

**Comment by berrg** [comment]
Ok, thanks for the feedback, I do now know the limitation of virtual scrollers. I will investigate a different option to see it is possible to make a scroller without large scroll area. Will report back later....

**Comment by pdanpdan** [comment]
Yes, please, it would help.
Because right now the only solution I can think of is a synthetic scroll (not native)

Advantages:
- unlimited size

Disadvantages:
- low precision, that does not map to the normal feel of scroll
- no native support, so things like mouse scroll, touch scroll, kbd scroll would need to be simulated
